pkg/kubelet/config/config.go

```diff
@@ -116,7 +116,7 @@ func (c *PodConfig) Restore(path string, updates chan<- interface{}) error {
 	var err error
```

Comment:
Not the issue for this commit, but can you do it like this (to remove the intent)? 
```golang
if c.checkpointManager != nil {
  return nil
}

c.checkpointManager, err := checkpointmanager.NewCheckpointManager(path)
if err != nil {
  return err
}

pods, err := checkpoint.LoadPods(c.checkpointManager)
if err != nil {
  return err
}

updates <- kubetypes.PodUpdate{Pods: pods, Op: kubetypes.RESTORE, Source: kubetypes.ApiserverSource}
return nil
```